lib/streamlit/DeltaGenerator.py

```diff
@@ -747,7 +746,9 @@ def exception(self, element, exception, exception_traceback=None):
         Parameters
         ----------
         exception : Exception
-            The exception to display.
+            The exception to display. If this is a StreamlitException,
+            then its message can contain markdown, which will then be
+            nicely formatted on the frontend.
```

Comment:
Remove the doc about `StreamlitException`. This pydoc is for user consumption, and they don't care about internal details like `StreamlitException`

Also it's `StreamlitAPIException`, no?